### ISSUE TITLE
[Meta] Image-minimizer wrong behaviour

### ISSUE THREAD

**Issue by yashpalgoyal1304** [TeamNewPipe/NewPipe#9469]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

https://github.com/TeamNewPipe/NewPipe/blob/aad5e26f31d52566a0b74b73d3aaa531a4695d7a/.github/workflows/image-minimizer.js#L85-L91

#### Current
* The image resizer github-actions bot resizes by fixing the height. 

#### Problem
* This is problematic when the available width is very small e.g. in portrait orientation (common in smart phones) or when the browser window is small-ed
* ... as in that case, the available width is very less, so, images get less width in their viewerport, but since their height is fixed - which is more than the required height at that available viewerport width, the image gets stretched in height

#### Proposal
* so, proposal is to use fixed width, which will satisfy both the scenarios - landscape/widescreen, as well as portrait
* i.e. if height is more, then reducing width _as specified_ in attribute will automatically reduce height as well, but if width gets reduced _externally_, then too, height will be automatically reduced per the available width.
* in other words, it can be thought like this: the height is never constrained physically - there's always scroll bars (making it virtually limitless), but there can always be constraint for width (since no scroll bars)

example: 

[<img alt="image" src="https://user-images.githubusercontent.com/19423063/204102249-f9adf7b0-cec2-45c2-9b4f-0ebf7aeec863.png" height=600 />](https://github.com/TeamNewPipe/NewPipe/issues/7375#issuecomment-1073190990)

**Comment by yashpalgoyal1304** [comment]
I faced this problem a lot in https://github.com/circuitikz/circuitikz/issues/665#issuecomment-1314504223, and more recently at https://git.bubu1.eu/Bubu/fdroidclassic/-/issues/37#note_952, and surprisingly the fix was very simple, so, i thought to file it here.
